backend/init_database.js

```diff
@@ -0,0 +1,58 @@
+const { debug } = require("console");
+const fs = require("fs");
+
```

Comment:
once folks start using ur code everyone is going to have a different version of `database.json` (b/c they'll all be created different data),  this will create all sorts of conflicts (not to mention we could accidentally delete the *actual database* if/when we deployed the site by committing && pulling a test database), so we want to avoid versioning this file, to do that add it to the `.gitignore` file

however, this means that for other folks, the first time they run ur code it will throw an error (b/c the `database.json` file won't exist for them) so u're going to want to add something like this to ur file:

```js
// a global variable w/the absolute path to the data base
const dbpath = `${__dirname}/database.json`
// checks to see if the database exists, if not it creates it
if (!fs.existsSync(dbpath)) {
  fs.writeFileSync(dbpath, '[]')
}
```